app/src/main/java/fr/free/nrw/commons/category/GridViewAdapter.java

```diff
@@ -98,13 +98,10 @@ public View getView(int position, View convertView, ViewGroup parent) {
      * @param author
      */
     private void setAuthorView(Media item, TextView author) {
-        if (!TextUtils.isEmpty(item.getCreator())) {
-            String uploadedByTemplate = getContext().getString(R.string.image_uploaded_by);
```

Comment:
I'd say the problem was this execution branch never set the visibility of the author back to visible

Reply:
Yeah. I got it. I was doing wrong. So, rather than making the visibility GONE can we set the textview blank?

Reply:
I think the empty field looks worse than the hidden field. As a user I would look at it and be confused whereas if it is hidden I might not even notice that it was missing

Reply:
Okay. I guess then in the TextUtils.isempty() portion, we first need to make the author textview visible

Reply:
>I'd say the problem was this execution branch never set the visibility of the author back to visible

👍